### ISSUE TITLE
26.0.2 not yet released

### ISSUE THREAD

**Issue by xgp** [keycloak/keycloak-client#98]
### Describe the bug

`26.0.2` jar not yet released to maven central, despite `26.0.2` release of keycloak

### Version

26.0.2

### Expected behavior

This should be automated along with the release of keycloak

### Actual behavior

No tag or release.

### How to Reproduce?

See https://github.com/keycloak/keycloak-client/tags

**Comment by benalbrecht** [comment]
i believe a new release is warranted due to #94

**Comment by jonkoops** [comment]
This is expected. The lifecycle of the clients is now separated from the main release, see the [release notes](https://www.keycloak.org/2024/10/keycloak-2600-released#_dedicated_release_cycle_for_the_client_libraries) for Keycloak 26.

**Comment by benalbrecht** [comment]
are you sure?

from the release notes of 26.0.4:
[#34284](https://github.com/keycloak/keycloak/issues/34284) Keycloak-admin-client should work with the future versions of Keycloak server admin/client-java

but there is no existing release of keycloak-admin-client that includes this piece of code.

so when i try to use the latest keycloak-client (26.0.1) with keycloak 26.0.4, i get the following error:

```
Execution error (UnrecognizedPropertyException) at com.fasterxml.jackson.databind.exc.UnrecognizedPropertyException/from (UnrecognizedPropertyException.java:61).
Unrecognized field "bruteForceStrategy" (class org.keycloak.representations.idm.RealmRepresentation), not marked as ignorable 
```

**Comment by jonkoops** [comment]
This is certainly a bug, but is not related to the release cycle specifically. This issue was fixed under #94 and needs a release, but that is unrelated to this issue that states the release cycle should match that of Keycloak. I'll ask the team if we can get a release out quickly.

**Comment by mposolda** [comment]
FYI. Keycloak-client 26.0.2 is released. It is compatible with Keycloak server 26.0.4. The issue with `Unrecognized field "bruteForceStrategy" (class org.keycloak.representations.idm.RealmRepresentation)` should be fixed.
